airflow/providers/google/cloud/operators/life_sciences.py

```diff
@@ -0,0 +1,74 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Operators that interact with Google Cloud Life Sciences service."""
+
+from typing import Iterable, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.providers.google.cloud.hooks.life_sciences import LifeSciencesHook
+from airflow.utils.decorators import apply_defaults
+
+
+class LifeSciencesRunPipelineOperator(BaseOperator):
+    """
+    Runs a pipeline
```

Comment:
Can we add a little more description, I know it is obvious but still. "Runs a Life Sciences Pipeline" or something